go.mod

```diff
@@ -18,26 +20,28 @@ require (
 	github.com/mgutz/ansi v0.0.0-20200706080929-d51e80ef957d
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/pkg/browser v0.0.0-20240102092130-5ac0b6a4141c
-	github.com/rivo/tview v0.0.0-20240406141410-79d4cc321256
+	github.com/rivo/tview v0.0.0-20241103174730-c76f7879f592
```

Comment:
Not sure why we need to update all these deps. Can we only update  libs related to this pr? Upgrading libs like tview could have unintended issues and requires proper testing.